internal/image/authorization.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/mail"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/hacbs-contract/ec-cli/internal/kubernetes"
+)
+
+var kubernetesClientCreator = kubernetes.NewClient
+
+// there can be multiple sign off sources (git commit, tag and jira issues)
+type signOffGetter interface {
```

Comment:
I thought we had agreed to move off of the "sign off" terminology altogether and use "authorize/authorization" instead to better convey the intent.

Reply:
This interface was just forgotten about and I'll fix. I did leave the source file under image since it's what we're authorizing.